src/modules/drawer.scss

```diff
@@ -109,6 +112,21 @@ html:not(.ua-ie-edge):not(.ua-ie-11) body {
     // #####
     // End of Header Items
     // #####
+    // Sticky Drawer
+    &.drawer-sticky {
+        bottom: 0;
+        top: unset;
+        .drawer-container { 
+            bottom: 0;
+            top: unset;       
+            > div:first-child {
+                border-top-left-radius: 5px;
+            }
+        }
+        .shadow_container > div { 
+            box-shadow: -9px 0 17px 0 rgba(0, 0, 0, 0.1) !important;
```

Comment:
Let's add the `sass-lint:disable-line no-important` comment on this line.

Reply:
Fixed